packages/client/get-port.ts

```diff
@@ -0,0 +1,9 @@
+import penumbra from '.';
+
+export const getAnyPenumbraPort = async (request?: boolean) => {
```

Comment:
What's the word `Any` imply here? Could we have multiple at once and it doesn't matter? Also it seems unused.